cfme/cloud/stack.py

```diff
@@ -236,7 +236,10 @@ def exists(self):
     def delete(self):
         """Delete the stack from detail view"""
         view = navigate_to(self, 'Details')
-        view.toolbar.configuration.item_select('Remove this Orchestration Stack', handle_alert=True)
+        msg = "Remove this Orchestration Stack"
+        if self.appliance.version >= '5.9':
+            msg = '{} from Inventory'.format(msg)
```

Comment:
**REQUIRED**: I would prefer the two strings in full, so we can grep for them. This we can't grep for when they change it again. In all the places.